README.md

```diff
@@ -3,3 +3,91 @@
 [![Build Status](https://travis-ci.com/single-spa/single-spa-layout.svg?branch=master)](https://travis-ci.com/single-spa/single-spa-layout)
 
 Layout engine for single-spa applications
+
+## Project Goal
+Single-spa layout is trying to solve the following problems:
```

Comment:
would just say `solves` or `attempts to solve` so we don't have to make a new PR to update this in the future :D